core/utils/poly-look/src/visualisations/charts/bubble-charts/bubbleCluster/bubbleCluster.js

```diff
@@ -49,17 +53,21 @@ export class BubbleCluster extends Chart {
     width,
     height,
     bubbleColor = defaultBubbleColor,
+    strokeColor = defaultStrokeColor,
     textColor = defaultTextColor,
+
     opacity = defaultOpacity,
-    showValues = defaultValueShowing,
+    text = defaultText,
     onBubbleClick = defaultOnClickFunction,
     filter,
   }) {
     super({ selector, data, width, height });
     this._bubbleColor = bubbleColor;
+    this._strokeColor = strokeColor;
     this._textColor = textColor;
+
     this._opacity = opacity;
-    this._showValues = showValues;
+    this._text = text;
```

Comment:
I'm confused here. This attribute will have a function as value?

Reply:
Still my changes - I believe all the changes in `BubbleCluster` where from my recent PR. But yes, it's just like how D3 works, we're not abstracting this logic away in our charts, but rather deliberately exposing it (for things to be more flexible).